pubspec.yaml

```diff
@@ -4,27 +4,29 @@ version: 0.12.0
 repository: https://github.com/johnpryan/flutter_map
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
   flutter: ">=2.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  tuple: ^1.0.2
-  latlong: ^0.6.1
+  tuple: ^2.0.0
+  latlong2: ^0.8.0
   positioned_tap_detector_2: ^1.0.0
-  transparent_image: ^1.0.0
+  transparent_image: ^2.0.0
   async: ^2.1.0
-  flutter_image: ^3.0.0
-  vector_math: ^2.0.0
+  flutter_image:
+    git: 
+      url: "https://github.com/flutter/flutter_image"
+      ref: null-safety
+  vector_math: ^2.1.0
   proj4dart: ^1.0.4
```

Comment:
Use this: https://github.com/maRci002/proj4dart/tree/migrate-null-safety for now

Reply:
done